### PR TITLE
Post-Garnett Cleanup

### PR DESCRIPTION
**Pull request by Ap0c**
## Why are you doing this?

To tidy up the mess I made while implementing Garnett. This merges the changes from the `garnett.scss` stylesheet back into the main JSX and CSS files. It also removes whacky things like background-imaging the logo while hiding the old logo, this can all be done in HTML (technically JavaScript) now.

**Note:** The fonts are still being loaded through the `garnett.scss` file (which is horrible) because we haven't come up with a font loading solution for them yet.

## Changes

- Replaced the old titlepiece logo SVG with the new logo SVG.
- Changed header background colour to match dotcom.
- Added the full Garnett palette to `colours.scss` and replaced magic number hex colours with the palette versions.
- Applied changes to the header to the `simpleHeader` component.
- Permanently removed the "contribute or subscribe" heading from the landing page.
- Merged the Garnett style changes into the bundles landing page files.
- Stopped including the `CrossProduct` component, rather than just hiding it with CSS.
- Dropped the video caption rather than just hiding it.

## Screenshots

header-change

### CODE REVIEW

File: assets/components/headers/simpleHeader/simpleHeader.scss [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-		}
+      height: 78px;
+    }
 
 		@include mq($from: mobileLandscape) {
-			height: 67px;
+      height: 85px;
 			padding-left: $gu-h-spacing;
 			padding-right: $gu-h-spacing;
 		}
```
Comment (svillafe) [inline assets/components/headers/simpleHeader/simpleHeader.scss]: I like the new indentation, Can you indent the entire file with the same one?

File: assets/pages/bundles-landing/bundlesLanding.scss [hunk 2]
```diff
@@ -526,7 +525,7 @@
       color: gu-colour(neutral-1);
 
       h3 {
-        color: gu-colour(guardian-brand);
+        color: gu-colour(sport-garnett-main-1);
```
Comment (svillafe) [inline assets/pages/bundles-landing/bundlesLanding.scss]: Do we want to keep the garnett name? I would change it to `sport-main-1`

File: assets/pages/bundles-landing/bundlesLanding.scss [hunk 3]
```diff
@@ -545,13 +544,13 @@
...
   .bundles--stacked {
-    background-color: gu-colour(guardian-brand-dark);
+    background-color: gu-colour(garnett-neutral-3);
     .bundles__cross-product {
       margin-top: 0;
       border-top: 0;
       padding-top: $gu-v-spacing;
     }
-    padding-bottom: $gu-v-spacing * 3;
+    padding-bottom: 0;
```
Comment (svillafe) [inline assets/pages/bundles-landing/bundlesLanding.scss]: Can you remove this `padding-bottom`?

File: assets/pages/bundles-landing/bundlesLanding.scss (hunk 2 above)
Comment (Ap0c) [reply assets/pages/bundles-landing/bundlesLanding.scss]: It's necessary for two reasons:

1. We're still using the old palette colours alongside, and they all use these names too.
2. This is what they're called in the main palette on dotcom.

**Comment by Ap0c** [comment]
Looks ok.

**Comment by Ap0c** [comment]
SIgh, PayPal ^.

**Pull request merged**